globals_test.go

```diff
@@ -299,6 +299,46 @@ func TestLoadGlobals(t *testing.T) {
 				),
 			},
 		},
+		{
+			name:   "stack with globals referencing globals using try",
+			layout: []string{"s:stack"},
+			globals: []globalsBlock{
+				{
+					path: "/stack",
+					add: globals(
+						str("field", "value"),
+						expr("field_try", `try(global.field, "error")`),
```

Comment:
not sure we should support this case... hmm... we could ofc.... it's not supported by terraform and in current implementation it will always be true or error with unknown var...

Reply:
I'm stupid, lets focus on tf behavior